cms/templates/js/video/transcripts/messages/transcripts-choose.underscore

```diff
@@ -18,7 +18,11 @@
     <% _.each(html5_list, function(value, index) {
         var type = grouped_list[value][0].type,
             file_name = value + ((type) ? ('.' + type) : ''),
-            message = interpolate(gettext("Timed Transcript from %(filename)s"), { filename: file_name }, true);
```

Comment:
pybabel_underscore mistakenly extracts `gettext` rather than the actual quoted string when it is written in such form.